docs/getting-started.md

```diff
@@ -489,58 +493,111 @@ real UK population*"
 
    A new `input.csv` file will be created in the `output` folder. Open that
    file (by left-clicking the filename in Visual Studio Code's Explorer, or
-   software like Excel). This time, you'll see it now synthetic data: an age
-   for 1000 randomly generated patients.
+   software like Excel). This time, you'll see it now contains synthetic data: an age
+   for 1000 randomly generated patients (we'll see shortly how this is defined).
 
 ### Add a chart
 
 **Every** study starts with a *study definition* like the one you just edited.
 When executed, a study definition generates a CSV of patient data.
 
-A real analysis will have several further steps after this. Each step can be
-written in [any of the programming languages supported in
+A real analysis will have several further steps after this. Each step is defined 
+in a separate file, and can be written in [any of the programming languages supported in
 OpenSAFELY](actions-scripts.md). In this tutorial, we're going to draw a
-histogram of ages, using four lines of Python.
+histogram of ages, using either four lines of Python or just a few more lines of R.
```

Comment:
It wasn't immediately obvious to me how you'd combine different languages, so the same example in python and R felt like it'd be useful

Reply:
I could make it 4 lines of R if brevity wins in this context?

Reply:
But I like having the two options!

Reply:
I mean, this is really the limit of my R abilities.  If you would like to tell me the 4 lines of R, I'd be happy to update it.  I could make it 4 lines of R, but only by removing line breaks :)